vendor_skins/Evercast/CI/install/osx/app2dmg.sh

```diff
@@ -0,0 +1,2 @@
+hdiutil create /tmp/tmp.dmg -ov -volname "EBS_2.5_Install" -fs HFS+ -srcfolder "./EBS.app" 
```

Comment:
I think as long as we know every place that has a version number we're fine.

Reply:
I don't know, I kind of like it.  One less thing to have to deal with when putting together a release.